testnets/airchainstestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "airchainstestnet",
+  "assets": [
+    {
+      "description": "The native staking token of Airchains Testnet.",
+      "denom_units": [
+        {
+          "denom": "amf",
+          "exponent": 0
+        },
+        {
+          "denom": "amf",
+          "exponent": 6
+        }
+      ],
+      "base": "amf",
+      "name": "Airchains",
+      "display": "amf",
+      "symbol": "AMF",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/airchainstestnet/images/airchains-mono.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/airchainstestnet/images/airchains-mono.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/airchainstestnet/images/airchains.png",
```

Comment:
Please do not add logotypes or combined logos. Only logomarks/icons and token logos.